### PR TITLE
Add more output variables

### PR DESCRIPTION
**Pull request by LukGerman**
## What

Added task execution role arn, task definition arn and task definition family to the outputs.

## Why

To provide ability to use those resources outside of module.

### CODE REVIEW

**Comment by maartenvanderhoef** [comment]
Try to do your pull requests with the following setup so it's clear for everyone why the changes are suggested and what they do.

## Why

## What

File: outputs.tf [hunk 1]
```diff
@@ -9,3 +9,15 @@ output "ecs_taskrole_name" {
 output "lb_target_group_arn" {
   value = "${module.alb_handling.lb_target_group_arn}"
 }
+
+output "task execution role arn" {
+  value = "${module.iam.ecs_task_execution_role_arn}"
+}
+
+output "aws_ecs_task_definition_arn" {
+  value = "${module.ecs_task_definition.aws_ecs_task_definition_arn}"
```
Comment (maartenvanderhoef) [inline outputs.tf]: should be `value = "${module.ecs_task_definition_selector.selected_task_definition_for_deployment}"`

File: outputs.tf [hunk 2]
```diff
@@ -9,3 +9,15 @@ output "ecs_taskrole_name" {
 output "lb_target_group_arn" {
   value = "${module.alb_handling.lb_target_group_arn}"
 }
+
+output "task execution role arn" {
```
Comment (maartenvanderhoef) [inline outputs.tf]: Not sure how this passed travis fmt, have you ran the new code yourself ?

File: outputs.tf (hunk 1 above)
Comment (LukGerman) [reply outputs.tf]: changed

File: outputs.tf (hunk 2 above)
Comment (LukGerman) [reply outputs.tf]: fixed

Comment (maartenvanderhoef) [reply outputs.tf]: I see spaces here `output "task execution role arn" {`

Comment (LukGerman) [reply outputs.tf]: Spaces were removed, please look at `Files changed` tab.

Comment (maartenvanderhoef) [reply outputs.tf]: My bad!

**Comment by maartenvanderhoef** [comment]
LG, thanks!

**Pull request merged**